lib/dialects/mssql/query-generator.js

```diff
@@ -354,7 +354,7 @@ var QueryGenerator = {
       if (attribute.type.values && !attribute.values) attribute.values = attribute.type.values;
 
       // enums are a special case
-      template = 'VARCHAR(10) NULL' /* + (attribute.allowNull ? 'NULL' : 'NOT NULL') */;
+      template = 'VARCHAR(255) NULL' /* + (attribute.allowNull ? 'NULL' : 'NOT NULL') */;
       template += ' CHECK (' + attribute.field + ' IN(' + Utils._.map(attribute.values, function(value) {
```

Comment:
If I add the checkConstraint to DataTypes, will it have access to attribute.values? 
Also, the checkConstraint relies on escape function which is in query-generator.js, do you want me to copy that over there? it will duplicate it :(  or move it out to another file and both files can require it in